src/box/lua/net_box.lua

```diff
@@ -390,6 +391,24 @@ local function new_sm(uri, opts)
                     })
                 end
             end
+            if not opts.fetch_schema and features.space_and_index_names then
+                remote.space = setmetatable({_remote = remote}, {
+                    __index = function(self, key)
+                        local space = {id = key, name = key,
+                                       _remote = self._remote,
+                                       _format_cdata =
+                                            box.internal.new_tuple_format()}
+                        space.index = setmetatable({space = space,
```

Comment:
There shouldn't be such thing as `space.index.space` unless there's an index named "space".

Reply:
Fixed.